docs/publish/maintain-breaking-changes.md

```diff
@@ -30,6 +30,16 @@ If you do encounter an app compatibility issue, App Assure engineers will work w
 
 To learn more about App Assure, watch [Bring your apps to Microsoft Edge with App Assure: tips and tricks](https://techcommunity.microsoft.com/t5/video-hub/bring-your-apps-to-microsoft-edge-with-app-assure-tips-and/ba-p/2167619). To submit your request for app compatibility with App Assure, complete the [Microsoft FastTrack Registration form](https://aka.ms/AppAssureRequest) or send an email to [achelp@microsoft.com](mailto:achelp@microsoft.com).
 
+## Deprecation policy
```

Comment:
What do you think about merging this with the office.js section above? It largely overlaps, so folding them together might read better?

Reply:
If we don't merge them, this says we might deprecate over 2 years, that says we generally won't, so I think we need to better align those.